### PR TITLE
Refactor dns and linodes page and clean up dns styles

### LINKED ISSUE #1072: Fix up DNS list

### PR DESCRIPTION
**Pull request by eatonphil**
screen shot 2017-01-27 at 11 54 04 am
screen shot 2017-01-27 at 11 54 17 am

Closes #1072

### CODE REVIEW

File: scss/components/PrimaryTable.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.PrimaryTable {
+    td {
+        vertical-align: middle;
+        line-height: initial;
+    }
+
+    &-row {
+        height: 65px;
+        border: 1px solid #ddd;
```
Comment (mparke) [inline scss/components/PrimaryTable.scss]: color var?

File: scss/components/PrimaryTable.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        margin: 0 15px;
+
+        &--groupLabel {
+            color: $light-black;
+            border: 0;
+            font-size: $font-small;
+            height: auto;
+
+            td {
+                padding-left: 0 !important;
```
Comment (mparke) [inline scss/components/PrimaryTable.scss]: can we avoid the `!important`?

File: scss/components/PrimaryTable.scss (hunk 1 above)
Comment (eatonphil) [inline scss/components/PrimaryTable.scss]: Oh whoops , this was for debugging. Will fix

File: src/dnsmanager/layouts/IndexPage.js [hunk 3]
```diff
@@ -51,50 +60,52 @@ export class IndexPage extends Component {
...
     const renderZones = () => (
-      <section>
-        <table>
-          <thead>
-            <tr>
-              <th>&nbsp;</th>
-              <th>Name</th>
-              <th>Type</th>
-              <th>&nbsp;</th>
+      <table className="PrimaryTable">
```
Comment (mparke) [inline src/dnsmanager/layouts/IndexPage.js]: is it time for a `List` component?

File: scss/components/PrimaryTable.scss (hunk 2 above)
Comment (eatonphil) [inline scss/components/PrimaryTable.scss]: It lost to https://github.com/linode/manager/blob/master/scss/manager.scss#L355

File: src/dnsmanager/layouts/IndexPage.js (hunk 3 above)
Comment (eatonphil) [inline src/dnsmanager/layouts/IndexPage.js]: Maybe, but I don't think the overhead is worth it because the content is too varied.

Comment (mparke) [inline src/dnsmanager/layouts/IndexPage.js]: styles and markup are the same, only data varies. I will plan to abstract it as part of #892

**Comment by eatonphil** [comment]
thanks!

**Pull request merged**